test/mysql/index.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const mysql = require("mysql");
+
+const promisify = async (method, thisArg, ...args) => {
+  new Promise((resolve) => {
```

Comment:
You forgot to return the promise here and the calls aren't serialized in the appmap.